lib/src/utils/index.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { getSectionName } from ':/utils/index';
+
+describe('Utils functions must work well to ensure a proper functioning of the library', () => {
+  it('should extract section name from widget id', async () => {
+    expect(getSectionName('root_section_widget')).toBe('section');
+  });
+
+  it('should extract section name from section id', async () => {
+    expect(getSectionName('root_section', true)).toBe('section');
+  });
+
+  it('should return undefined when the id is not valid', async () => {
+    expect(getSectionName('root_section')).toBeUndefined();
+  });
+
+  it('should return undefined if the id is root', async () => {
+    expect(getSectionName('root')).toBeUndefined();
+  });
```

Comment:
Those tests does not use asynchrone logic so there is no need to declare async function.
```suggestion
  it('should extract section name from widget id', () => {
    expect(getSectionName('root_section_widget')).toBe('section');
  });

  it('should extract section name from section id', () => {
    expect(getSectionName('root_section', true)).toBe('section');
  });

  it('should return undefined when the id is not valid', () => {
    expect(getSectionName('root_section')).toBeUndefined();
  });

  it('should return undefined if the id is root', () => {
    expect(getSectionName('root')).toBeUndefined();
  });
```